wezterm-gui/src/overlay/selector.rs

```diff
@@ -368,6 +403,7 @@ pub fn selector(
         always_fuzzy: args.fuzzy,
         args,
         event_name,
+        selection: String::new(),
```

Comment:
see other comments for context on this!

```suggestion
        selection: String::new(),
        labels: vec![],
        max_items: 0,  -- and remove the unused size and max_items local vars above
```

Reply:
the `max_items: 0` bit is here so that the logic to compute max_items and labels can be centralized to the same single location

Reply:
I have done this now (though I kept the order).